sync/class.jetpack-sync-module-posts.php

```diff
@@ -168,9 +174,6 @@ function filter_post_content_and_add_links( $post_object ) {
 
 		$post->permalink               = get_permalink( $post->ID );
 		$post->shortlink               = wp_get_shortlink( $post->ID );
-		$post->dont_email_post_to_subs = Jetpack::is_module_active( 'subscriptions' ) ?
```

Comment:
I am nervous about keeping it, since the value is often wrong, and there are also instances where we have the wrong jetpack version number in WPCOM, which could lead to the bug resurfacing. However I will restore it for now and we can make a separate PR.